rules/windows/persistence_suspicious_service_created_registry.toml

```diff
@@ -10,7 +10,7 @@ Identifies the creation of a suspicious ImagePath value. This could be an indica
 stealthily persist or escalate privileges through abnormal service creation.
 """
 from = "now-9m"
-index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*"]
+index = ["logs-endpoint.events.*"]
```

Comment:
The integration is only parsing DWORD and QWORD registry values, so registry.data.strings will always be null here. Dropping Support